test/libethereum/blockchain.cpp

```diff
@@ -124,632 +85,425 @@ void doBlockchainTests(json_spirit::mValue& _v, bool _fillin)
 				else
 					importBlockNumber++;
 
-				//each time construct a new blockchain up to importBlockNumber (to generate next block header)
-				vBiBlocks.clear();
-				vBiBlocks.push_back(biGenesisBlock);
-
-				TransientDirectory tdBc;
-				TransientDirectory tdStateDB;
-				FullBlockChain<Ethash> bc(rlpGenesisBlock.out(), AccountMap(), tdBc.path(), WithExisting::Kill);
-
-				//OverlayDB database (State::openDB(td_stateDB.path(), h256{}, WithExisting::Kill));
-				State state(OverlayDB(State::openDB(tdStateDB.path(), h256{}, WithExisting::Kill)), BaseState::Empty); //= importer.m_statePre;
-				ImportTest::importState(o["pre"].get_obj(), state);
-				state.commit();
-
-				//import previous blocks
-				for (size_t i = 1; i < importBlockNumber; i++) //0 block is genesis
+				if ((string)boost::unit_test::framework::current_test_case().p_name != "bcWalletTest")
```

Comment:
Still need to fix that problem. Because this just happens to be the only big BlockChainTest, but may be there will be reasons for other rather large BlockChainTests